doc/rst/language/spec/domains.rst

```diff
@@ -1010,20 +1010,21 @@ domains only.
 
    proc domain.dim(d: int): range
 
-Returns the range of indices described by dimension ``d`` of the domain.
+Returns the range of indices described by dimension ``d`` of the domain,
+where ``d`` is a value from ``0`` to ``rank-1``.
 
    *Example*.
 
    In the code 
 
    .. code-block:: chapel
 
-      for i in D.dim(1) do
-        for j in D.dim(2) do
+      for i in D.dim(0) do
+        for j in D.dim(1) do
           writeln(A(i,j));
 
-   domain ``D`` is iterated over by two nested loops. The first
-   dimension of ``D`` is iterated over in the outer loop. The second
+   domain ``D`` is iterated over by two nested loops. The initial
+   dimension of ``D`` is iterated over in the outer loop. The other
```

Comment:
I see the reason for this change and I am fine with it. But it wouldn't bother me if we were to use "first" to refer to `dim(0)`